org-gtd-files.el

```diff
@@ -48,16 +48,21 @@ This is the inbox. Everything goes in here when you capture it.
   "Return the full path to the inbox file."
   (org-gtd--path org-gtd-inbox))
 
-(defun org-gtd--inbox-file ()
-  "Create or return the buffer to the GTD inbox file."
-  (let ((file-path (org-gtd--path org-gtd-inbox)))
-    (if (f-file-p file-path)
-        (find-file-noselect file-path)
-      (with-current-buffer (find-file-noselect file-path)
-        (insert org-gtd-inbox-template)
-        (org-mode-restart)
-        (basic-save-buffer)
-        (current-buffer)))))
+;;;###autoload
+(defun org-gtd-find-inbox ()
+  "Make the GTD inbox buffer current, creating the file if necessary.
+Returns the GTD inbox buffer, e.g. for use with
+`with-current-buffer', but is mostly intended as a function
+target in Org capture templates."
+  (let* ((path (org-gtd-inbox-path))
+         (exists (file-exists-p path)))
```

Comment:
I don't like using `let*` unless I absolutely need to, and here I don't think it gets us anything to use it; it forces the reader to pay extra attention to the variable bindings to figure out how they are related, and yes, there's only two right now, but what guarantee that it will stay that way?